### ISSUE TITLE
Tracker -> All issues or anywhere with a table with selection checkboxes no "Select All" button

### ISSUE THREAD

**Issue by gnud** [hcengineering/platform#6863]
### Description of the issue
I can see something on the list header to the left of "Backlog" title something resembling a check box, and assuming it is a select all checkbox, however is not clicable, however looking from the inspector I can see it's some svg icon to represent a status.

### Your environment
* Version of huly: Using latest cloud version
* Browser: Firefox 130.0
* Ubuntu (not applicable for the cloud version)

### Steps to reproduce
Open any list in the Tracker and there is no "Select All" option

### Expected behaviour
If  this feature exists should be able to select all the items at once, if not I'd like to request a new feature

### Actual behaviour
No "Select All" visibly in the page

**Comment by eshubhat** [comment]
Indeed, Select all option is not given and the SVG component must not be the part of the feature "Select All", as even sub topics under the main topic has that svg icon but are not used for selection

My proposed Solution would be to add a select All button with the same ui similar to the checkbox given in the sub topics
please do assign it to me
